code/parse/sexp.cpp

```diff
@@ -30138,6 +30137,12 @@ const char *sexp_error_message(int num)
 		case SEXP_CHECK_MISPLACED_SPECIAL_ARGUMENT:
 			return "Found " SEXP_ARGUMENT_STRING " without an argument handler higher in the sexp node tree";
 
+		case SEXP_CHECK_AMBIGUOUS_GOAL_NAME:
```

Comment:
Instead of "ambiguous", how about "duplicate"? IMO that more precisely captures the error.

"Ambiguous" is a bit, well, ambiguous, hence the need for the parenthetical explanation.

Reply:
A duplicate name is not itself an error; it only becomes an error if `is-event-true` could refer to multiple events.  It's the ambiguity that's the real problem here.

In line with that reasoning, I think the error message says everything that needs to be said.

Reply:
Ok, sure.